src/routes/FollowListPopup.svelte

```diff
@@ -100,7 +100,7 @@
 										>{user.iq}</span
 									>
 								</div>
-								<p class="text-sm text-muted-foreground">@{user.handle}</p>
+								<p class="text-sm text-muted-foreground"><a href="https://lyntr.com/@{user.handle}">@{user.handle}</a></p>
```

Comment:
It works but please change href to `/@{user.handle}`

Reply:
Ok, I hope I understood the fix correctly, is this bc it uses a hardcoded URL prefix?

Reply:
> Ok, I hope I understood the fix correctly, is this bc it uses a hardcoded URL prefix?

Yeah because on localhost it will redirect to lyntr.com